.readthedocs.yaml

```diff
@@ -12,7 +12,7 @@ build:
   jobs:
     post_checkout:
       - git fetch --tags --depth 1 # Also fetch tags
-      - git describe               # Useful for debugging
+      - git describe --always      # Useful for debugging
```

Comment:
what's this?

Reply:
Few recent PRs, like this one and PR #692, were failing to build docs because `git describe` command is not finding a tag to describe. This is a workaround.

Reply:
But this is a problem that we need to fix some other way; if the `git describe` failed then the version of rockcraft on the docs will be wrong

Reply:
But what is causing it to fail all of a sudden?

Reply:
I don't know, nothing changed on our side as far as I can tell

Reply:
I fixed this on `main`, you can undo this change now

Reply:
done